File: helium-java/src/integrationtest/java/com/heliumhq/Environment.java
package com.heliumhq;

import static com.heliumhq.api_impl.application_context.DevelopmentAPIConfig.
		getHeliumFile;

public class Environment {

	public static String getIntegrationtestResource(String relPath) {
		return getHeliumFile("src/integrationtest/resources/" + relPath);
	}

	public static String getITFileURL(String page) {
		return pathToFileURL(getIntegrationtestResource(page));
	}

	private static String pathToFileURL(String path) {
		return "file:///" + path.replace("\\", "/");
	}

}

File: helium-python/src/integrationtest/python/helium_integrationtest/environment.py
from os import path
from os.path import abspath, pardir

def get_integrationtest_resource(*rel_path_compontents):
	return abspath(path.join(
		__file__, pardir, pardir, pardir, pardir, pardir, pardir,
		'src', 'integrationtest', 'resources', *rel_path_compontents
	))

def get_helium_package_source(*rel_path_compontents):
	return abspath(path.join(
		__file__, pardir, pardir, pardir, pardir, pardir,
		'src', 'main', 'python', *rel_path_compontents
	))

def path_to_file_url(path):
	return 'file:///' + path.replace('\\', '/')

def get_it_file_url(page):
	return path_to_file_url(get_integrationtest_resource(page))